src/jit/codegencommon.cpp

```diff
@@ -7102,10 +7119,10 @@ void CodeGen::genSetGSSecurityCookie(regNumber initReg,
 
         *pInitRegZeroed = false;
 
+#if CPU_LOAD_STORE_ARCH
         //  mov   reg, dword ptr [compiler->gsGlobalSecurityCookieAddr]
```

Comment:
the comment actually applies to the "else" case, and the "if" case needs a new comment (that was never written) to describe how it generates code.